build/Nuget/Microsoft.VisualStudio.ProjectSystem.VSInsertionNuget.proj

```diff
@@ -1,26 +1,49 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <ImportGroup Label="Settings">
-    <Import Project="..\Targets\Settings.targets" />
-  </ImportGroup>
+  <Import Project="..\Targets\Settings.targets" />
+  
   <PropertyGroup>
     <NuGetVersion>1.0.0.0</NuGetVersion>
     <NuGetVersionType>InsertionPurposeOnly</NuGetVersionType>
+    <NuGetOutDir>$(OutDir)NuGet\$(NuGetVersionType)\</NuGetOutDir>
   </PropertyGroup>
-  <ItemGroup Label="Redist">
-    <NuSpec Include="Microsoft.VisualStudio.ProjectSystem.VSInsertion.nuspec" />
+  
+  <ItemGroup>
+    <NuSpec Include="Microsoft.VisualStudio.ProjectSystem.VSInsertion.nuspec">
+      <NuPkgOutput>$(NuGetOutDir)%(FileName).$(NuGetVersion).nupkg</NuPkgOutput>
+    </NuSpec>
+
+    <!-- Make note, without actually knowing what's included in the NuSpec, 
+         we just assume that everything in the output dir is, and treat them
+         as inputs so that we rerun packaging if they change. 
+         Long term we'll want to use a packaging project, such as NuProj where 
+         the NuSpec is auto-generated based on the inputs. -->
+    <PackageAssets Include="$(OutDir)\*.*" Exclude="*.log" />
   </ItemGroup>
-  <Target Name="Build" Inputs="@(NuSpec)" Outputs="@(NuSpec-&gt;NuGet\$(NuGetVersionType)\%(Identity).$(NuGetVersion).nupkg">
+
+  <ItemGroup>
+    <NuPkgOutput Include="@(NuSpec -> '%(Filename).nupkg'" />
```

Comment:
Ah, yes, it's no longer needed due to the NuPkgOutput metadata, I'll remove it. The braces don't need to match - that's the syntax for transforms: https://msdn.microsoft.com/en-us/library/ms171476.aspx.